.github/workflows/test_on_push.yml

```diff
@@ -71,12 +88,26 @@ jobs:
           brew update
           brew install graphviz
           brew install openblas
+          brew bundle dump --force
+
+      # Experimental solution for caching Homebrew installs
+      - name: Cache MacOS dependencies
+        uses: actions/cache@v3
+        if: matrix.os == 'macos-latest'
+        with:
+          # paths for Homebrew packages, formulae, casks, and binaries
+          path: |
+            /usr/local/Homebrew/*
+            /usr/local/Cellar/*
+            /usr/local/etc/*
+            /usr/local/opt/*
+          key: brew-cache-${{ runner.os }}-${{ hashFiles('**/Brewfile.lock.json') }}
```

Comment:
> We can probably remove the Homebrew caching altogether since it does not offer any real benefits?

Yep, we can get rid of this then. I am guessing the one minute speedup is due to the homebrew environment flags and not this step?

Reply:
Though, by "a very minute speedup", I was referring to "minute" as in _small_, not that it caused a one-minute speedup 😆

Reply:
Ah, that makes it clear, thanks!